tempCodeRunnerFile.py

```
@@ -0,0 +1,54 @@
+def simulate_growth(start_date_str):
+    planting_date = datetime.strptime(start_date_str, "%Y-%m-%d")
```

Ruff (0.12.2)

Undefined name `datetime`

(F821)